ui/src/message_popup/renderers/audio_response.jsx

```diff
@@ -20,11 +20,13 @@ export default React.createClass({
     question: React.PropTypes.object.isRequired,
     scaffolding: React.PropTypes.object.isRequired,
     limitMs: React.PropTypes.number.isRequired,
+    elapsedMs: React.PropTypes.number.isRequired,
```

Comment:
Yeah I added it in since it's being passed in now that `PopupQuestion` manages that time and uses the same set of props with each response component.  I think it's doesn't super matter, but this seems slightly better since `elapsedMs` is information that is available to a response component (and the audio component just happens not to use it).  Could go either way though.